acceptance/bundle/telemetry/deploy-config-file-count/script

```diff
@@ -1,9 +1,12 @@
 trace $CLI bundle deploy
 
-trace cat out.requests.txt | jq 'select(has("path") and .path == "/telemetry-ext") | .body.protoLogs[] | fromjson'
+# Ensure the file is fully written
+sync
```

Comment:
Does sync work on windows?

Reply:
It does but we don't really need it anymore

Reply:
I think we do, otherwise what does a separate file accomplish?

Reply:
We use now `>` which ensure that the entire output of the input on the left is flushed to the file (`out.telemetry.txt`) before it completes